src/Microsoft.ML.ImageAnalytics/VectorToImageTransform.cs

```diff
@@ -139,17 +401,27 @@ private sealed class ColInfoEx
             public readonly Single Scale;
             public readonly bool Interleave;
 
-            public bool Alpha { get { return (Colors & ColorBits.Alpha) != 0; } }
-            public bool Red { get { return (Colors & ColorBits.Red) != 0; } }
-            public bool Green { get { return (Colors & ColorBits.Green) != 0; } }
-            public bool Blue { get { return (Colors & ColorBits.Blue) != 0; } }
+            public bool Alpha { get { return (Colors & ImagePixelExtractingEstimator.ColorBits.Alpha) != 0; } }
```

Comment:
>public bool Alpha { get { return (Colors & ImagePixelExtractingEstimator.ColorBits.Alpha) != 0; } } [](start = 12, length = 99)

Not a big deal, but if you wanted to bring this transplanted code up to "modern standards" by using `=>` style syntax to make this a bit less crazy taht would be fine with me.